### ISSUE TITLE
Resolve nightly scan failures

### ISSUE THREAD

**Issue by ADPennington** [raft-tech/TANF-app#2766]
**Description:**
The nightly scan has not been running successfully for at least the last 3 months. It has not run successfully in prod since April 2023. 

_develop insights_
![develop_nightly](https://github.com/raft-tech/TANF-app/assets/63075587/7d66fb44-58bf-48b7-9cf9-a9adea1b8f14)

_latest backend report in develop_
here: https://app.circleci.com/pipelines/github/raft-tech/TANF-app?branch=develop
![nightly_scandec2023](https://github.com/raft-tech/TANF-app/assets/63075587/f5c1c884-fe34-40bb-bd3d-9bff528934bd)

Findings need to be reviewed to determine if these are legitimate and if so, resolved. Else flagged or ignored as false positives. 

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] Nightly scan workflow should be successful (for backend and frontend)
- [ ] Any fixes should be tested in a deployed environment to capture any new findings before this ticket closes
- [ ] Non-TDP URLs should be excluded, where possible. 
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Investigate findings
- [ ] Run Testing Checklist and confirm all tests pass

**Comment by ADPennington** [comment]
decided during backlog today to unblock the zap scans from being saved in the backend when there are findings that cause the pipeline to fail. @jtimpe will create a new ticket for this.

**Comment by robgendron** [comment]
@jtimpe is this refined as is? If so, please remove the placeholder tasks, notes, supporting documents, and questions.

**Comment by andrew-jameson** [comment]
Closed upon review, solved in PR linked.
